synfig-studio/plugins/lottie-exporter/common/DashItemList.py

```diff
@@ -0,0 +1,131 @@
+"""
+DashItemList.py
+Will store the Parameters class for Dash item list
+"""
+
+import sys
+import common
+from common.DashItem import DashItem
+from synfig.animation import to_Synfig_axis
+sys.path.append("..")
+
+
+class DashItemList:
+    """
+    Class to store Synfig format Dash items list
+    """
+    def __init__(self, dash_item, parent):
+        """
+        This parent should be parameter
+        """
+        self.parent = parent
+        self.dash_item = dash_item
+        self.type = dash_item.attrib["type"]
+        self.entry_list = []
+        self.extract_entries(self.entry_list)
+
+    def get(self):
+        """
+        Returns the original param
+        """
+        return self.dash_item
+
+    def __getitem__(self, itr):
+        """
+        Returns the child corresponding to itr
+        """
+        return self.entry_list[itr]
+
+    def __setitem__(self, itr, val):
+        """
+        Sets the value of child corresponding to itr
+        """
+        self.entry_list[itr] = val
+
+    def get_layer_type(self):
+        """
+        Recursively go till the top until layer is not reached
+        """
+        if isinstance(self.parent, common.Layer.Layer):
+            return self.parent.get_type()
+        return self.parent.get_layer_type()
+
+    def get_layer(self):
+        """
+        Recursively find the layer of this bline
+        """
+        if isinstance(self.parent, common.Layer.Layer):
+            return self.parent
+        return self.parent
+
+    def get_type(self):
+        """
+        Returns the type of bline
+        """
+        return self.type
+
+    def get_loop(self):
+        """
+        Returns whether the bline is looped or not
+        """
+        loop = False
```

Comment:
This can be simplified:
```
        if "loop" in self.dash_item.keys():
            v = self.dash_item.attrib["loop"]
            return v == "true"

        return False
```